### PR TITLE
Add developer resources into member resources

### LINKED ISSUE #583: Add developer resources page into member resources
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Context for documentation change

We get recurring questions that have been answered. And we also received valuable resources from our members in our Slack channels.

### Proposed solution

Create a page—like "Developer Resources"—to compile the resources that both members and non-members can consume.
We can include this page in the Member Resources.

### Resources that can help

_No response_

### Collaborators

_No response_

### Code of Conduct

- [X] I've read the Code of Conduct and understand my responsibilities as a member of the Virtual Coffee community

### PR DESCRIPTION
**Pull request by adiati98**
* Add UndrawCollection svg

## Linked Issue

Closes #583 

## Description

- Add the developer resources page into member resources.
- Add an SVG.

## Code of Conduct

> By submitting this pull request, you agree to follow our [Code of Conduct](https://virtualcoffee.io/code-of-conduct/)

### CODE REVIEW

**Comment by adiati98** [comment]
Hi @danieltott,
As per our conversation with @BekahHW, I created the developer resources page.
I made a draft of the pull request for your preview.

Let me know what you think.
Thank you! 😊

File: app/routes/__frontend/resources/virtual-coffee/developer-resources.mdx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+<TextContainer background="light" showBackToTopLink={false}>
+
+## Hi friends 👋
+
+<LeadText>
+
+We received various questions and resources from our members every day. Some topics arose multiple times, and others were too valuable not to share.
+
+So, we are compiling resources dropped by our members in the Slack channels and other resources that we believe would be useful for you!
```
Comment (CBID2) [inline app/routes/__frontend/resources/virtual-coffee/developer-resources.mdx:21]: [Nitpick] Maybe you can hyperlink the Slack channels here @adiati98? 
```suggestion
So, we are compiling resources dropped by our members in the Slack channels and other resources that we believe would be useful for you!
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @adiati98! :) I don't know if you're still working on this pull request, but I gave a suggestion on how you can improve. I hope it helps! :)

**Comment by adiati98** [comment]
Hi @CBID2,

We're postponing this PR because we're thinking of a better form to compile and share the articles and resources.
But thank you for your suggestion! 😊

**Comment by CBID2** [comment]
> Hi @CBID2,
> 
> We're postponing this PR because we're thinking of a better form to compile and share the articles and resources. But thank you for your suggestion! 😊

Ok

File: app/routes/__frontend/resources/virtual-coffee/developer-resources.mdx (hunk 1 above)
Comment (BekahHW) [reply app/routes/__frontend/resources/virtual-coffee/developer-resources.mdx:21]: @CBID2 I know that we're currently postponing, but also wanted to say that bc the slack is invite-only and the resources are for the wider tech community, we wouldn't want to hyperlink to the channels.

**Pull request closed without merging**